autokeras/keras_layers.py

```diff
@@ -76,4 +76,5 @@ def get_config(self):
 
 CUSTOM_OBJECTS = {
     'MultiColumnCategoricalEncoding': MultiColumnCategoricalEncoding,
+    'IndexLookup': index_lookup.IndexLookup,
```

Comment:
Consider using `StringLookup` and `IntegerLookup` respectively, instead of `IndexLookup`. The latter will not be exposed in the API

Reply:
It seems they are only in 2.3. We are basing on 2.2 now. I will add an issue about this and about bump up to 2.3. We will add them when we upgrade to tf 2.3. Thanks.